packages/llvm/llvm.3.9/files/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+make=$1
+libdir=$2
+
+# Create build dir
+mkdir build
+cd build
+
+# Configure
+cmake ..
+#    -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON \
+#    -DLLVM_LINK_LLVM_DYLIB:BOOL=ON \
+#    ..
+
+# Build
+grep "^ocaml_llvm.*/fast:$" Makefile | tr ":\n" " " | xargs "$make"
```

Comment:
Awesome !!! Thanks a lot :)
However there is still ~2 more issues:
* The installation problem: ```cmake -P bindings/ocaml/cmake_install.cmake``` installs all the files in ```${prefix}/lib/ocaml``` instead of ```${prefix}/lib/llvm```
* The linking problem: well, it doesn't seems like llvm_libs contains LLVM-3.9 with the BUILD_DYNLIB/LINK_DYNLIB flags. But maybe I'm missing something here.

There is also an optional problem (as it was also present in the previous releases): the llvm_{arch} ocaml packages can't be reached as their META files doesn't have the right names. They have to be located in a separate directory (something like ```${prefix}/lib/llvm_{arch}```) or be merged to the main META file as a sub package (it would be a nicer solution I think).